index.d.ts

```diff
@@ -3365,6 +3366,16 @@ interface RaycastResult {
 	surfaceNormal: Vector3Mp
 }
 
+interface ConsoleMp {
+	logInfo(message: string, save: boolean, saveAsync: boolean): any;
```

Comment:
What about using `void` instead of `any` - this would be more definite?

Reply:
Fair enough